src/main/features/claim/routes/pay.ts

```diff
@@ -93,10 +95,10 @@ async function successHandler (res, next) {
       logger.error(`missing consent not given role for user, User Id : ${user.id}`)
     }
 
-    let features: string
-    if (await featureTogglesClient.isFeatureToggleEnabled(user, roles, 'cmc_admissions')) {
+    let features: string = ''
+    launchDarklyClient.callFeatureFlag(user, config.get<string>('launchDarkly.featureFlag-keys.admissions'), () => {
```

Comment:
Can we change it so that we just update the `isFeatureToggleEnabled` method to call the new LaunchDarklyClient so we only need to change one place in our code?

Reply:
the calling method is different I think, it takes a closure rather than responding with a boolean, so probably requires refactoring either way

Reply:
@timja is right ... a refactoring is needed either way.

Reply:
I've no doubt that it will require refactoring. I'm asking for that to be done inside the isFeatureToggleEnabled so that it will continue to return a boolean :)

Reply:
not sure if that is worth doing, you'll likely miss out on some of the SDK features if you wrap it in this way,